pkg/schema/v2/operations.go

```diff
@@ -0,0 +1,38 @@
+package schema
```

Comment:
mind adding a `docs.go` to each of these new packages, explaining what they do and how they are currently for testing only?